### PR TITLE
🍰 2019/kw25/improve_import_accuracy

### PR DESCRIPTION
**Pull request by Tirokk**
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-06-19T11:56:58Z" title="Wednesday, June 19th 2019, 1:56:58 pm +02:00">Jun 19, 2019</time>_
_Merged <time datetime="2019-06-24T11:03:24Z" title="Monday, June 24th 2019, 1:03:24 pm +02:00">Jun 24, 2019</time>_
---

## 🍰 Pullrequest

- Splits `contributions` and `follows` by referenced type and import them accordingly
- cleanup on neo4j import folder

### Issues

- relates #692

### Todo

- [X] None

### CODE REVIEW

File: deployment/legacy-migration/maintenance-worker/migration/mongo/export.sh [hunk 1]
```diff
@@ -24,9 +31,12 @@ ssh -4 -M -S my-ctrl-socket -fnNT -L 27018:localhost:27017 -l ${SSH_USERNAME} ${
 export_collection "badges"
 export_collection "categories"
 export_collection "comments"
-export_collection "contributions"
+export_collection_query "contributions" "{'type': 'DELETED'}" "DELETED"
```
Comment (Tirokk) [inline deployment/legacy-migration/maintenance-worker/migration/mongo/export.sh:34]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T20:52:12Z" title="Friday, June 21st 2019, 10:52:12 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,9 +31,12 @@ ssh -4 -M -S my-ctrl-socket -fnNT -L 27018:localhost:27017 -l ${SSH_USERNAME} ${
 export_collection "badges"
 export_collection "categories"
 export_collection "comments"
-export_collection "contributions"
+export_collection_query "contributions" "{'type': 'DELETED'}" "DELETED"
```


Apparently, the `type` got overwritten when sb. deleted a post or cando and we cannot distinguish it anymore. I would simply not import `DELETED` data at all.

Comment (Tirokk) [reply deployment/legacy-migration/maintenance-worker/migration/mongo/export.sh:34]: > [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-06-24T10:15:15Z" title="Monday, June 24th 2019, 12:15:15 pm +02:00">Jun 24, 2019</time>_

---


yes, i export them only - i do not import the deleted data.

**Comment by Tirokk** [comment]
> [codecov[bot]](https://github.com/apps/codecov) **Authored by [codecov[bot]](https://github.com/apps/codecov)**
_<time datetime="2019-06-24T10:16:06Z" title="Monday, June 24th 2019, 12:16:06 pm +02:00">Jun 24, 2019</time>_

---


# [Codecov](https://codecov.io/gh/Human-Connection/Human-Connection/pull/863?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@3c36cad`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

**Review by Mogge — APPROVED** [review]
Do I understand this correctly, that you take out the candos and follows pointing to organizations? If yes, I think I can approve this, hopefully you tested the import. Ideally you can test this on docker, too.

**Pull request closed without merging**